releases/tex/master/extra/crowbar-backup

```diff
@@ -0,0 +1,68 @@
+#!/bin/bash
```

Comment:
The name of the script is problematic, as we already have a crowbar-backup script in Cloud 5 :-)